uniflow/flow/config.py

```diff
@@ -1,62 +1,20 @@
-"""Model, Extract, Transform and Pipeline config module."""
+"""Extract, Transform and Pipeline config module."""
 
 from dataclasses import dataclass, field
-from typing import Dict, Optional
-from uniflow.op.prompt_schema import GuidedPrompt
+from typing import Optional
 
+from uniflow.op.prompt_schema import GuidedPrompt
+from uniflow.op.transform.model_config import (
+    HuggingfaceModelConfig,
+    ModelConfig,
+    NougatModelConfig,
+    OpenAIModelConfig,
+)
 
 ###########################################################
-#                  All Model Configs                      #
+#                   All Extract Configs                   #
 ###########################################################
 
-@dataclass
-class ModelConfig:
-    """Model Config Class."""
-
-    model_name: str = "gpt-3.5-turbo-1106"
-    model_server: str = "OpenAIModelServer"
-
-
-@dataclass
-class OpenAIModelConfig(ModelConfig):
-    """OpenAI Model Config Class."""
-
-    model_server: str = "OpenAIModelServer"
-    num_call: int = 1
-    temperature: float = 0.9
-    response_format: Dict[str, str] = field(default_factory=lambda: {"type": "text"})
-
-
-@dataclass
-class HuggingfaceModelConfig(ModelConfig):
-    """Huggingface Model Config Class."""
-
-    model_name: str = "mistralai/Mistral-7B-Instruct-v0.1"
-    batch_size: int = 1
-    model_server: str = "HuggingfaceModelServer"
```

Comment:
qq: where does all these config moved to?

Reply:
Moved to `uniflow/op/transform/model_config.py`